content/software-security/compliance/cmmc-2/cmmc-documentation.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Documentation Requirements for CMMC 2.0"
+description: "Documentation Requirements for CMMC 2.0"
+lead: "Learn about the different documentation requirements for each of CMMC 2.0's maturity levels. "
+type: "article"
+date: 2024-08-09T19:10:09+00:00
+lastmod: 2024-08-09T19:10:09+00:00
+contributors: ["Erin Glass"]
+draft: false
+tags: ["compliance", "CMMC 2.0", "standards"]
+images: []
+menu:
+  docs:
+    parent: "cmmc-2"
+weight: 004
+toc: true
+---
+
+The requirements for documentation under Cybersecurity Maturity Model Certification (CMMC) 2.0 vary by maturity level, with higher levels requiring more extensive documentation to demonstrate compliance. Here is a breakdown of the documentation requirements by maturity level:
+
```

Comment:
Much of this page feels like it overlaps with the Overview page, where the levels are first introduced, despite focusing here on documentation differences across levels. I wonder whether the two pages could be merged or otherwise streamlined for brevity.

Reply:
Merged, good idea!